ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -293,7 +292,7 @@ - (void)getUserVideo:(NSDictionary *)constraints
       }
     }
     NSArray * mandatoryKeys = [mandatory allKeys];
-    if([mandatoryKeys count]==0){
+    if([mandatoryKeys count] == 0){
```

Comment:
should be `if ([mandatoryKeys count] == 0) {`

space between `if` and `(` 
space between `)` and `{`